### PR TITLE
Fixes invisible Avians

### PR DESCRIPTION
**Pull request by JohnGinnane**
It seems that in @Carbonhell's anger... he killed them!

:cl: JohnGinnane
fix: Fixed Avians (and any other Hippie races) being invisible
/:cl:

Problem was here: https://github.com/HippieStation/HippieStation/commit/ce07ba7c5b13f998bcb569fe5924e7db495b55b0#diff-919aeecfd27738b229072d09c451cfb1L381

### CODE REVIEW

**Comment by Carbonhell** [comment]
not sure how i missed this, good catch

File: code/modules/surgery/bodyparts/bodyparts.dm [hunk 1]
```diff
@@ -378,6 +378,14 @@
 				limb.icon_state = "[species_id]_[body_zone]_[icon_gender]"
 			else
 				limb.icon_state = "[species_id]_[body_zone]"
+		// Hippie start - Hippie races (e.g. Avians)
```
Comment (Carbonhell) [inline code/modules/surgery/bodyparts/bodyparts.dm]: hippie should be lowercase

File: code/modules/surgery/bodyparts/bodyparts.dm [hunk 2]
```diff
@@ -378,6 +378,14 @@
...
+		if(should_draw_hippie)
+			limb.icon = 'hippiestation/icons/mob/mutant_bodyparts.dmi'
+			if(should_draw_gender)
+				limb.icon_state = "[species_id]_[body_zone]_[icon_gender]"
+			else
+				limb.icon_state = "[species_id]_[body_zone]"
+		// Hippie end
```
Comment (Carbonhell) [inline code/modules/surgery/bodyparts/bodyparts.dm]: Same

File: code/modules/surgery/bodyparts/bodyparts.dm (hunk 1 above)
Comment (Carbonhell) [reply code/modules/surgery/bodyparts/bodyparts.dm]: Also -- not -

File: hippiestation/code/modules/mob/living/carbon/human/species_types/avians.dm [hunk 3]
```diff
@@ -19,3 +19,6 @@
 /datum/species/bird/on_species_loss(mob/living/carbon/human/C, datum/species/new_species, pref_load)
 	C.draw_hippie_parts(TRUE)
 	. = ..()
+
+/datum/species/bird/check_roundstart_eligible()
```
Comment (Carbonhell) [inline hippiestation/code/modules/mob/living/carbon/human/species_types/avians.dm:23]: ?? I did not touch this, what is this about

**Review by Carbonhell — DISMISSED** [review]

File: code/modules/surgery/bodyparts/bodyparts.dm (hunk 1 above)
Comment (JohnGinnane) [reply code/modules/surgery/bodyparts/bodyparts.dm]: when did this change? Case sensitivity shouldn't be an issue if your search doesn't use it, i.e. using regex like `\/\/( |)hippie`

**Comment by JohnGinnane** [comment]
Hold on, IPC is missing

**Comment by JohnGinnane** [comment]
@Carbonhell this is ready

File: hippiestation/code/modules/mob/living/carbon/human/species_types/avians.dm (hunk 3 above)
Comment (JohnGinnane) [reply hippiestation/code/modules/mob/living/carbon/human/species_types/avians.dm:23]: It's now required I think, when I started the server I couldn't even select this race from the list during character creation

**Review by Carbonhell — APPROVED** [review]

**Pull request merged**